app/authentification/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
```

Comment:
`from django.contrib.auth import get_user_model` is a better way to refer to User. The reason is:
* https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#referencing-the-user-model

Reply:
True, definitely a better way.